test/dummy/db/schema.rb

```diff
@@ -972,7 +972,8 @@
   end
 
   create_table "news_tematica_news_tematicas", force: true do |t|
-    t.integer  "tematica_id"
+    t.integer  "suscribible_id"
+    t.string   "suscribible_type"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.